include/dali/core/tensor_shape.h

```diff
@@ -637,6 +638,43 @@ struct TensorListShapeBase {
     shapes.resize(num_samples * sample_dim);
   }
 
+
+  ///@{
```

Comment:
What is this line? Never seen that before

Reply:
It's doxygen's grouping ;)